depends/packages/libxcb_util_keysyms.mk

```
@@ -0,0 +1,31 @@
+package=libxcb_util_keysyms
+$(package)_version=0.4.0
+$(package)_download_path=https://xcb.freedesktop.org/dist
+$(package)_file_name=xcb-util-keysyms-$($(package)_version).tar.bz2
+$(package)_sha256_hash=0ef8490ff1dede52b7de533158547f8b454b241aa3e4dcca369507f66f216dd9
+$(package)_dependencies=libxcb_util
```

While looking at the `./configure` log, I think the correct dependencies are:
```suggestion
$(package)_dependencies=libxcb xproto
```